dotnet/src/Agents/UnitTests/AgentChatTests.cs

```diff
@@ -144,5 +144,16 @@ private sealed class TestAgent : ChatHistoryKernelAgent
 
             yield return new ChatMessageContent(AuthorRole.Assistant, "sup");
         }
+
+        public override async IAsyncEnumerable<StreamingChatMessageContent> InvokeStreamingAsync(
+            ChatHistory history,
+            [EnumeratorCancellation] CancellationToken cancellationToken = default)
+        {
+            await Task.Delay(0, cancellationToken);
```

Comment:
Why is this Delay here?

Reply:
To satisfy the `async` keyword.  A method with return type `IAsyncEnumerable` must be declared `async`.

(No race condition or other weirdness)

Reply:
Updated